spark/src/main/java/org/apache/iceberg/spark/SparkTableUtil.java

```diff
@@ -425,6 +455,21 @@ private static void importUnpartitionedSparkTable(SparkSession spark, TableIdent
       List<DataFile> files = TableMigrationUtil.listPartition(
           partition, Util.uriToString(sourceTable.location()), format.get(), spec, conf, metricsConfig, nameMapping);
 
+      if (checkDuplicateFiles) {
+        Dataset<Row> fileDs = spark.createDataset(
```

Comment:
fileD's -> importedPaths ? Just looking for a slightly more descriptive name here

Reply:
Actually you use existing files below which is a good name, so maybe importedFiles here?

Reply:
Done